parachain/pallets/control/src/lib.rs

```diff
@@ -33,7 +33,6 @@ pub mod pallet {
 	use super::*;
 	use frame_support::{pallet_prelude::*, traits::EnsureOrigin};
 	use frame_system::pallet_prelude::*;
-	use xcm::v3::MultiLocation;
```

Comment:
#890 Removes `xcm::prelude::*` above. If we merge both we'll probably end up with no `MultiLocation` here. Could we instead remove the prelude above?

Reply:
#890 is merged and this is rebased on top of that.